### ISSUE TITLE
Script command for setting the time zone

### ISSUE THREAD

**Issue by beltoforion** [Stellarium/stellarium#497]
**Is your feature request related to a problem? Please describe.**
There does not seem to be a way to set the local time zone from a script. A script can set the observer location but it does not seem to be able to also set the time zone. As a consequence whenever i display the local time, the script will display a value that may be incorrect in the context of the script. (If the geographic context of the script is haweii i'd like to see the local time of hawaii not mine)

**Describe the solution you'd like**
Two things would help me:
- a command for setting the time zone from the script
- a way to specify the time zone in a landscape files (LandscapeMgr.setFlagLandscapeSetsLocation(true) seems pointless if the time zone cannot be set too)

**Describe alternatives you've considered**
Only ever use "utc" in script, create a helper in javascript to compute local time on my own.

**More context:**
I have written an educational script for computing analemmas (https://github.com/beltoforion/stellarium_scripts/tree/master/scripts) The script takes the observer location as input parameters. Since i seem to be unable to set the time zone the display of the local time in the script is incorrect if stellarium is not set to the proper time zone for the location given by the script.

**Comment by gzotti** [comment]
Indeed, core.setObserverLocation() should add timezone as well. We improved timezones not too long ago, but missed this.

**Comment by gzotti** [comment]
I hope I caught all pitfalls in the increasingly complex location panel. 

New script commands:

-   getAllTimezoneNames()
-   setTimezone(QString tz)

and getObserverLocationInfo() provides two new items:
 - timezone
 - location-timezone

This reflects the fact that our location database has TZ entries stored for the location, but timezone can be overridden.

**Comment by beltoforion** [comment]
Amazing. Thanks alot!

**Comment by gzotti** [comment]
I forgot the other thing. Now you can specify something like
```
timezone = Europe/Berlin
```
in the location block in landscape.ini
